vm/object.go

```diff
@@ -7,6 +7,10 @@ import (
 var mainObj *RObject
 
 func init() {
```

Comment:
We shouldn't avoid use this `init` function and package level variables, I'll fix these issues in future commits.